static/includes/apps/SMBLimitations.md

```diff
@@ -0,0 +1,149 @@
+&NewLine;
+
+{{< expand "SMB Limitations" "v" >}}
+
+#### Windows, Linux, and MacOS Client SMB Limitations 
+{{< truetable >}}
+|  | Windows | Linux (Ext4) | Linux (XFS) | MacOS |
+|--|---------|--------------|-------------|-------|
+| Maximum file name length (characters) | 255 | 255 | 255 | 255 |
+| Maximum share name length (characters) | 255 | 255 | 255 | 255 |
+| Maximum length for path (characters) | 260* | 4096 | 4096 | 255*** |
+| Invalid characters | angle brackets < ><br>colon :<br>asterick *<br>slashes / \<br>pipe | | NULL (?0)<br>slash (/) | NULL (?0)<br>slash (/) | slash /<br>colon :<br> |
+| Case sensitivity | Yes | No | No | Yes |
+| Single file size/write size** | 16 TB | 16 TB | 8 EB | # |
```

Comment:
I don't know what relevance is here. The single file size is basically determined by the filesystem limitations of the server holding the file. I'm not aware of any SMB client imposing a filesize limitation.

Reply:
This was listed as information I should provide. If it is not relevant I can strip this out.